docs/content/nodes/archive-api-rate-limits.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Flow Archive Access API rate limits on Dapper Labs Archive nodes
+---
+
+#### Rate limits
+
+Following are the current rate limits for the [Archive Access Node gRPC API](archive-access-api) in total across all Dapper Labs Access nodes. The actual limits are enforced by each individual node. If you are using a load balanced endpoint, the limits below are the max rate, and may vary depending on how your requests are routed.
```

Comment:
"Archive Access Node gRPC API" sounds a bit of a mouthful. Maybe just 'Archive Node gRPC API' which we can explain is a subset of Access API?

Reply:
Agreed, your suggestion sounds more concise.